requirements.txt

```diff
@@ -0,0 +1,2 @@
+beautifulsoup4==4.9.1
+requests==2.31.0
```

Comment:
per the most recent discussion https://splunk.slack.com/archives/GBSGHUUD9/p1685364655311529 we should stop explicitly including requests in requirements.txt. this version of requests doesn't work on the most recent cloud release anyways